src/main/java/md/jack/task8/Op.java

```diff
@@ -0,0 +1,38 @@
+package md.jack.task8;
+
+
+/**
+ * Represents an operation that we will want to time.
+ * Added a static and default method to the interface
+ * that we had used earlier.
+ * <p>
+ * From <a href="http://courses.coreservlets.com/Course-Materials/">the
+ * coreservlets.com tutorials on JSF 2, PrimeFaces, Ajax, JavaScript, jQuery, GWT, Android,
+ * Spring, Hibernate, JPA, RESTful Web Services, Hadoop, Spring MVC,
+ * servlets, JSP, Java 8 lambdas and streams (for those that know Java already),
+ * and Java 8 programming (for those new to Java)</a>.
+ */
+
+@FunctionalInterface
+public interface Op
+{
+    static void timeOp(Op operation)
+    {
+        long startTime = System.nanoTime();
+        operation.runOp();
+        long endTime = System.nanoTime();
+        double oneBillion = 1_000_000_000;
+        double elapsedSeconds = (endTime - startTime) / oneBillion;
+        System.out.printf("  Elapsed time: %.3f seconds.%n", elapsedSeconds);
+    }
+
+    void runOp();
+
+    default Op combinedOp(Op secondOp)
+    {
+        return (() -> {
+            runOp();
```

Comment:
extract multi-time lambdas body in standalone method(use intelij hint extract to method reference)